### PR TITLE
Change factors using fct_recode from forcats

### PR DESCRIPTION
**Pull request by cgross95**
Addresses #385 to simplify the way that factors are recoded.

### CODE REVIEW

**Review by juanfung — CHANGES_REQUESTED** [review]
@cgross95 Awesome! Actually think this works quite well but maybe we could at least mention how to do it using factor? It doesn't even need to be a code chunk, just an inline comment!

**Comment by cgross95** [comment]
Sorry, I'm not actually sure how to change factors with `factor()`. Do you mean mentioning to the previous way using the `levels()` function to modify the `levels` attribute directly? Or is there another way?

**Comment by juanfung** [comment]
@cgross95 exactly! Really appreciate your work on this issue!

File: _episodes_rmd/02-starting-with-data.Rmd [hunk 1]
```diff
@@ -404,12 +404,18 @@ informative than integers because factors are self describing: `"cement"`,
...
 
 ```{r, results = TRUE, purl = FALSE}
 levels(respondent_floor_type)
 
-levels(respondent_floor_type)[2] <- "brick"
+respondent_floor_type <- fct_recode(respondent_floor_type, brick = "cement")
+
+## as an alternative, we could change the "cement" level directly using the
+## levels() function, but we have to remember that "cement" is the second level
+# levels(respondent_floor_type)[2] <- "brick"
```
Comment (cgross95) [inline _episodes_rmd/02-starting-with-data.Rmd:418]: @juanfung Is something like this what you had in mind?

**Review by juanfung — APPROVED** [review]
yes works for me!

**Comment by juanfung** [comment]
@cgross95 thank you for your responsiveness and for handling this issue!

**Comment by juanfung** [comment]
@cgross95 awesome, you did a great job and the help is most appreciated!

**Pull request merged**